README.md

```diff
@@ -79,14 +79,31 @@ We currently tested and provide explicit support for the following WiFi adapters
 
 ## Contributing
 
-To contribute send github pull requests targeting [this](https://github.com/resin-os/meta-resin) repository.
+### Waffle.io
+
+resin-os repositories use [https://waffle.io/resin-os/meta-resin](https://waffle.io/resin-os/meta-resin) to manage its issues (including PRs). We define there the issues flow and automatic rules to assign labels.
+
+### Development flow
+
+To contribute send github pull requests targeting [this](https://github.com/resin-os/meta-resin) or any [resin-os](https://github.com/resin-os) repository. Here are the steps you need to follow:
+
+1. Always work on a github issue - create one or make sure you have one in a resin-os repository.
+2. Create a branch using the following naming scheme:
+  * If the issue you are referencing is in the repository you are pushing to use the following branch namng scheme `<issue number>-<branch name>`. Example: 123-bug-fix.
+  * If the issue you are referencing is not in the repository you are pushing to, use the following branch naming scheme: `<username>/<repo>#<issue number>-<branch name>`. Example: resin-os/meta-resin#123-bug-fix.
```

Comment:
This is to take advantage of a waffle feature that would move issues on in progress when a branch is created: https://github.com/waffleio/waffle.io/wiki/FAQs#branch-moving